code/game/machinery/fat_sucker.dm

```diff
@@ -215,5 +215,10 @@
 		return
 	start_at = 100
 	stop_at = 0
-	to_chat(user, "<span class='notice'>You remove the access restrictions and lower the automatic ejection threshold!</span>")
+	balloon_alert(user, "restrictions lifted")
+	addtimer(CALLBACK(src, .proc/finish_emag_act, user), 0.5 SECONDS)
```

Comment:
Remove `finish_emag_act`, and do:

```suggestion
	addtimer(CALLBACK(src, .proc/balloon_alert, user, "automatic ejection threshold lowered"), 0.5 SECONDS)
```

If this doesn't work, try `/atom/proc/balloon_alert`.